src/build.cc

```diff
@@ -164,10 +164,12 @@ void BuildStatus::BuildFinished() {
 }
 
 string BuildStatus::FormatProgressStatus(
-    const char* progress_status_format) const {
+    const char* progress_status_format,
+    const string to_print) const {
```

Comment:
I can change it to const string&. I do not think any differences as string passed to to_string is not too long as it could cause some performance changes...